src/kOS/Suffixed/Direction.cs

```diff
@@ -160,6 +160,16 @@ private void DirectionInitializeSuffixes()
             return new Vector(a.Rotation * (Vector3d)b);
         }
 
+        public static Vector operator +(Direction a, Vector b)
+        {
+            return new Vector(a.Rotation * (Vector3d)b);
+        }
+
+        public static Vector operator +(Vector b, Direction a)
+        {
+            return new Vector(a.Rotation * (Vector3d)b);
+        }
+
```

Comment:
This is mostly for backwards compatibility.  Technically I should have supported _every_ operator between vectors and directions because the old code basically read "if other value is a vector, regardless of operator, apply the rotation to the vector".  I noticed there are a couple of random scripts out there that use the `+` operator.  I don't mind deleting this, but we'll need to add the breaking tag.